package.json

```diff
@@ -13,7 +13,7 @@
     "@replit/database": "^2.0.1",
     "cheerio": "^1.0.0-rc.5",
     "discord.js": "^12.5.3",
-    "dotenv": "^8.2.0",
+    "dotenv": "^8.6.0",
```

Comment:
Why do we need an update here?

Reply:
No, when I did the install in my VS code is updated that.